Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -3448,9 +3448,9 @@ homeScreenContent push state =  let
   where 
     contentView state = 
       linearLayout
-      [ width MATCH_PARENT
-      , height WRAP_CONTENT
-      ][ 
+      [ width $ V $ screenWidth unit
+      , height $ V 160
```

Comment:
Is this working on different screen sizes? Also Sometimes images adjust their width according to the image height as well

Reply:
Tested by increasing screen size also. working fine.
<img width="1728" alt="Screenshot 2024-07-02 at 09 56 26" src="https://github.com/nammayatri/nammayatri/assets/85025190/acd4f3d6-d9a3-4ba7-817c-d82857693669">